VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -9,6 +9,12 @@
 #include "../VoodooI2CController/VoodooI2CControllerDriver.hpp"
 #include "VoodooI2CDeviceNub.hpp"
 
+#include <Availability.h>
+
+#ifndef __ACIDANTHERA_MAC_SDK
+#error "This kext SDK is unsupported. Dowload from https://github.com/acidanthera/MacKernelSDK"
```

Comment:
Minor typo here

Reply:
It's meant to be a preprocessor error to ensure that the MacKernelSDK is present. 

TBH, I rather not have it as a global include and have it fail when attempting to build the implementation (cpp file). The error is quite straightforward and easy to understand.

Reply:
Dowload → Download ^^